client/src/pages/SignUp/SignUp.tsx

```diff
@@ -40,10 +42,12 @@ export default function Register(): JSX.Element {
   return (
     <Grid container component="main" className={classes.root}>
       <CssBaseline />
-      <Grid item xs={12} sm={8} md={7} elevation={6} component={Paper} square>
+      <Grid item xs={12} sm={8} md={6} elevation={0} component={Paper}>
+        <img src={BG_img} style={{ height: '100%' }} />
```

Comment:
Same thing here, also, is there any way we could make this not repeated?

Reply:
Okay, I will give a chnage